src/python/export_module_manager.cpp

```diff
@@ -53,10 +65,32 @@ void export_module_manager(py_module_reference m) {
       .def("run_as",
            [](py_obj self, py_obj pt, py_obj key, pybind11::args args) {
                return self.attr("at")(key).attr("run_as")(pt, *args);
-           });
+           })
+      .def("keys", &ModuleManager::keys)
+      .def("__getitem__", [](ModuleManager& self, const type::key& key) {
+          return self.at(key);
+      });
+    // Python iterator doesn't work, yet. Gives "double free or corruption
+    // (out)" error. .def(
+    //   "begin",
+    //   [](ModuleManager& self) {
+    //       return pybind11::make_iterator(self.begin(), self.end());
+    //   },
+    //   pybind11::keep_alive<0, 1>())
+    // .def(
+    //   "end",
+    //   [](ModuleManager& self) {
+    //       return pybind11::make_iterator(self.end(), self.end());
+    //   },
+    //   pybind11::keep_alive<0, 1>())
+    // .def(
+    //   "__iter__",
+    //   [](ModuleManager& self) {
+    //       return pybind11::make_iterator(self.begin(), self.end());
+    //   },
+    //   pybind11::keep_alive<0, 1>());
```

Comment:
I'd avoid checking in unworking code. It's better to open an issue describing what you were trying to do, and to include the code there.

Reply:
Done.